examples/src/cifar10_cnn_big.cpp

```diff
@@ -0,0 +1,64 @@
+//=======================================================================
+// Copyright (c) 2016 Baptiste Wicht
+// Distributed under the terms of the MIT License.
+// (See accompanying file LICENSE or copy at
+//  http://opensource.org/licenses/MIT)
+//=======================================================================
+
+#define ETL_COUNTERS
+#define ETL_GPU_TOOLS
```

Comment:
I do not think this is necessary. You can remove it from each of the three examples.